eth2near/finality-update-verify/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "finality-update-verify"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+eth-types = { path ="../../contracts/near/eth-types/", features = ["eip1559"]}
+bls = { git = "https://github.com/aurora-is-near/lighthouse.git", branch = "wasm-tmp", default-features = false, features = ["milagro"]}
+eth2-client  = { path ="../../contracts/near/eth2-client"}
```

Comment:
It seems, as only `utility` mod is used from `eth2-client`, we need to use `utility` crate explicitly.

Reply:
Merged `the-merge` branch and fixed in 7a4de8829fb6e854a2a9f41165aa6a0eddb3bba1